common/enablers/enableEpochsHandler.go

```diff
@@ -39,6 +41,330 @@ func (handler *enableEpochsHandler) EpochConfirmed(epoch uint32, _ uint64) {
+	case core.DelegationSmartContractFlag:
+		return epoch >= handler.enableEpochsConfig.DelegationSmartContractEnableEpoch
+	case core.DelegationSmartContractFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.DelegationSmartContractEnableEpoch
+	case core.CorrectLastUnJailedFlagInSpecificEpochOnly:
+		return epoch == handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
+	case core.CorrectLastUnJailedFlag:
+		return epoch >= handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
+	case core.RelayedTransactionsV2Flag:
+		return epoch >= handler.enableEpochsConfig.RelayedTransactionsV2EnableEpoch
+	case core.UnBondTokensV2Flag:
+		return epoch >= handler.enableEpochsConfig.UnbondTokensV2EnableEpoch
+	case core.SaveJailedAlwaysFlag:
+		return epoch >= handler.enableEpochsConfig.SaveJailedAlwaysEnableEpoch
+	case core.ReDelegateBelowMinCheckFlag:
+		return epoch >= handler.enableEpochsConfig.ReDelegateBelowMinCheckEnableEpoch
+	case core.ValidatorToDelegationFlag:
+		return epoch >= handler.enableEpochsConfig.ValidatorToDelegationEnableEpoch
+	case core.IncrementSCRNonceInMultiTransferFlag:
+		return epoch >= handler.enableEpochsConfig.IncrementSCRNonceInMultiTransferEnableEpoch
+	case core.ESDTMultiTransferFlag,
+		core.ESDTNFTImprovementV1Flag:
+		return epoch >= handler.enableEpochsConfig.ESDTMultiTransferEnableEpoch
+	case core.GlobalMintBurnFlag:
+		return epoch < handler.enableEpochsConfig.GlobalMintBurnDisableEpoch
+	case core.ESDTTransferRoleFlag:
+		return epoch >= handler.enableEpochsConfig.ESDTTransferRoleEnableEpoch
+	case core.BuiltInFunctionOnMetaFlag,
+		core.TransferToMetaFlag:
+		return epoch >= handler.enableEpochsConfig.BuiltInFunctionOnMetaEnableEpoch
+	case core.ComputeRewardCheckpointFlag:
+		return epoch >= handler.enableEpochsConfig.ComputeRewardCheckpointEnableEpoch
+	case core.SCRSizeInvariantCheckFlag:
+		return epoch >= handler.enableEpochsConfig.SCRSizeInvariantCheckEnableEpoch
+	case core.BackwardCompSaveKeyValueFlag:
+		return epoch < handler.enableEpochsConfig.BackwardCompSaveKeyValueEnableEpoch
+	case core.ESDTNFTCreateOnMultiShardFlag:
+		return epoch >= handler.enableEpochsConfig.ESDTNFTCreateOnMultiShardEnableEpoch
+	case core.MetaESDTSetFlag:
+		return epoch >= handler.enableEpochsConfig.MetaESDTSetEnableEpoch
+	case core.AddTokensToDelegationFlag:
+		return epoch >= handler.enableEpochsConfig.AddTokensToDelegationEnableEpoch
+	case core.MultiESDTTransferFixOnCallBackFlag:
+		return epoch >= handler.enableEpochsConfig.MultiESDTTransferFixOnCallBackOnEnableEpoch
+	case core.OptimizeGasUsedInCrossMiniBlocksFlag:
+		return epoch >= handler.enableEpochsConfig.OptimizeGasUsedInCrossMiniBlocksEnableEpoch
+	case core.CorrectFirstQueuedFlag:
+		return epoch >= handler.enableEpochsConfig.CorrectFirstQueuedEpoch
+	case core.DeleteDelegatorAfterClaimRewardsFlag:
+		return epoch >= handler.enableEpochsConfig.DeleteDelegatorAfterClaimRewardsEnableEpoch
+	case core.RemoveNonUpdatedStorageFlag:
```

Comment:
missing check for `FixOOGReturnCodeEnableEpoch`? 
```GO
handler.setFlagValue(epoch >= handler.enableEpochsConfig.FixOOGReturnCodeEnableEpoch, handler.fixOOGReturnCodeFlag, "fixOOGReturnCodeFlag")
```

Reply:
check exists on [L355](https://github.com/multiversx/mx-chain-go/blob/new_methods_enable_epochs_handler/common/enablers/enableEpochsHandler.go#L355), flag is `FixOOGReturnCodeFlag`

Reply:
right, missed it due to the changed position